WcaOnRails/app/views/registrations/psych_sheet_event.html.erb

```diff
@@ -30,19 +30,19 @@
           <td class="pos <%= registration.tied_previous ? "tied-previous" : "" %>">
             <%= registration.pos %>
           </td>
-          <td class="name"><%= registration.name %></td>
+          <td class="name"><%= registration.select_name %></td>
```

Comment:
`registration.name` seems better, is there any reason for using `select_` prefix here?